_posts/2019-09-12-this-week-in-solid.html

```diff
@@ -0,0 +1,78 @@
+---
+permalink: /newsletter/2019-09-12
+layout: article
+title: This Week in Solid 2019-09-12
+breadcrumbs:
+  - title: "Solid Newsletter"
+    url: /newsletter
+  - title: "This Week in Solid 2019-09-12"
+---
+
+<p>Hello and welcome to another edition of This Month in Solid!</p>
+<p><a href="https://solidproject.org">Solid</a> is a set of open specifications, built on existing open standards, that describes how to build applications in such a way that users can conveniently switch between data storage providers and application providers and take the data generated along.</p>
+<p>This is a monthly summary of Solid progress.</p>
+<p>Want something mentioned? Contact the Solid Manager at solid+info@theodi.org. This Month in Solid is openly developed <a href="https://github.com/solid/solidproject.org/blob/main/_posts/newsletter/next.md">on GitHub</a>. If you find any errors in this month's issue please <a href="https://github.com/solid/solidproject.org/pulls">submit a pull request</a>.</p>
```

Comment:
Please replace "Solid Manager" with "Open Data Institute" and link to https://theodi.org/insights/projects/odi-and-solid-building-a-future-where-data-works-for-everyone/ on the words open data institute. This term is also used on pages on the website with higher traffic such as the community page.

Since these paragraphs turn up in all of the newsletters I wonder if it is worth templating?